src/renderer/components/+extensions/install.tsx

```diff
@@ -29,6 +29,7 @@ import { SubTitle } from "../layout/sub-title";
 import { TooltipPosition } from "../tooltip";
 import { ExtensionInstallationStateStore } from "./extension-install.store";
 import { observer } from "mobx-react";
+import { MaterialTooltip } from "../+catalog/material-tooltip/material-tooltip";
```

Comment:
Already mentioned about this before but why `MaterialTooltip` is under `+catalog` if it's general purpose component?

Reply:
It was created in `+catalog` by mistake. Moved to `./components` just now.